pep-0012.rst

```diff
@@ -50,44 +50,36 @@ To use this template you must first decide whether your PEP is going
 to be an Informational or Standards Track PEP.  Most PEPs are
 Standards Track because they propose a new feature for the Python
 language or standard library.  When in doubt, read PEP 1 for details
-or contact the PEP editors <peps@python.org>.
+or contact a PEP editor for assistance.
```

Comment:
Is there a documented way to do this that could be linked to here?

Reply:
(Mariatta also asked about this, will keep discussion to that thread)